pom.xml

```diff
@@ -816,6 +818,30 @@
       </properties>
     </profile>
     <profile>
+      <id>skip-node-tests</id>
+      <activation>
+        <property>
+          <name>skipTests</name>
+          <value>true</value>
+        </property>
+      </activation>
+      <properties>
+        <skip.node.tests>--skipTests</skip.node.tests>
+      </properties>
+    </profile>
+    <profile>
+      <id>skip-node-lint</id>
+      <activation>
+        <property>
+          <name>skipLint</name>
```

Comment:
I would suggest just making this be activated by `skipTests` too, which we have sort of been overloading to mean “skip anything which does not contribute to the build product” (cf. `skip-findbugs-with-tests`, `skip-javadoc-with-tests`).